src/gui/editors/PianoRoll.cpp

```diff
@@ -3533,7 +3533,8 @@ void PianoRoll::selectAll()
 
 	for( const Note *note : m_pattern->notes() )
 	{
-		int len_ticks = note->length();
+		int len_temp = note->length();
+		int len_ticks = len_temp > 0 ? len_temp : 1;
 
```

Comment:
The select function finds 0 length a bit rude so we make them 1. Don't know if this is a good minimum length though but it seem to work just fine.